src/components/ProfileCard/index.tsx

```diff
@@ -62,12 +71,13 @@ const ProfileCard: React.FC<Props> = ({ profile }: Props) => {
         {profile.technologies?.length > 0 && (
           <ul className="flex flex-wrap items-center justify-start">
```

Comment:
if we are not going to use the `li` inside the `ul` maybe we can transform it to a regular `div`. Not sure what are the accessibility implcaitions.